setup.py

```diff
@@ -305,7 +305,7 @@ def run(self):
     'toolbar = spyder.plugins.toolbar.plugin:Toolbar',
     'tours = spyder.plugins.tours.plugin:Tours',
     'variable_explorer = spyder.plugins.variableexplorer.plugin:VariableExplorer',
-    'frames_explorer = spyder.plugins.framesexplorer.plugin:FramesExplorer',
```

Comment:
Please move this line to be before

    'editor = spyder.plugins.editor.plugin:Editor'

That's because we try to keep this list in alphabetical order.